ledger/catchpointwriter_test.go

```diff
@@ -441,7 +442,7 @@ func TestFullCatchpointWriterOverflowAccounts(t *testing.T) {
 	const maxResourcesPerChunk = 5
 	testWriteCatchpoint(t, ml.trackerDB().Rdb, catchpointDataFilePath, catchpointFilePath, maxResourcesPerChunk)
 
-	l := testNewLedgerFromCatchpoint(t, catchpointFilePath)
+	l := testNewLedgerFromCatchpoint(t, catchpointFilePath, ml.trackerDB().Rdb)
```

Comment:
Swap order of last two args, so it matches `testWriteCatchpoint`?

Reply:
@jannotti https://github.com/algorand/go-algorand/pull/4818/commits/32529a07c77263c1b8bb3b596a4b90122ece9089 reorders arguments to bring in the feedback.